migrations-shelley/1573472706108_add-known-legacy-addresses-table.js

```diff
@@ -0,0 +1,38 @@
+/* eslint-disable camelcase */
+
+const fs = require('fs');
+const readline = require('readline');
+const SqlString = require('sqlstring');
+
+exports.shorthands = undefined;
+
+const SQL = `
+CREATE TABLE known_legacy_addresses  (
+  address text,
+  PRIMARY KEY (address)
+);
+`
+
+exports.up = async (pgm, run) => {
+
+  pgm.sql(SQL)
+
+  const fileStream = fs.createReadStream('migrations-shelley/data/known-legacy-addresses-byron-mainnet.txt')
+  const rl = readline.createInterface({
+    input: fileStream,
+    crlfDelay: Infinity
+  })
+
+  for await (const line of rl) {
+    // ignore comments
+    if (line && !line.startsWith('#')) {
+      pgm.sql('insert into known_legacy_addresses (address) values ({line})', { line: SqlString.escape(line) })
```

Comment:
```
pgm.sql('insert into known_legacy_addresses (address) values ({line})', { line: SqlString.escape(line) })
```

can be changed to:
```
pgm.sql('insert into known_legacy_addresses (address) values ($1)', strippedString)
```

Reply:
Discussed with @crchemist-ip - migrations don't support that syntax here, so we still need to use *some* "external" escaping here. But we replaced `sqlstring` with building a big single query with `squel` with the escaping built-in.